ReadingRainbow/api-functions/authentication.js

```diff
@@ -0,0 +1,39 @@
+import * as SecureStore from 'expo-secure-store';
+
+export async function RetrieveToken(iUsername, iPassword) {
+
+    console.log(JSON.stringify({ username: iUsername, password: iPassword })); //Test purposes only
+    const encodedUsername = encodeURIComponent(iUsername);
+    const encodedPassword = encodeURIComponent(iPassword);
+
+    console.log('APi username: ', encodedUsername);
+    console.log('APi password: ', encodedPassword);
+
+    //const fullurl = `https://localhost:5001/api/token?username=${encodedUsername}&password=${encodedPassword}`;
+    //const fullurl = 'https://localhost:5001/api/token';
+    //173.16.48.206

```

Comment:
If these are not needed/incorrect calls, go ahead and remove them.

Reply:
This is from commit one, they were removed in commit 2.